java/vector/src/test/java/org/apache/arrow/vector/ipc/TestArrowStream.java

```diff
@@ -64,7 +82,7 @@ public void testStreamZeroLengthBatch() throws IOException {
     try (IntVector vector = new IntVector("foo", allocator);) {
       Schema schema = new Schema(Collections.singletonList(vector.getField()));
       try (VectorSchemaRoot root =
-             new VectorSchemaRoot(schema, Collections.singletonList(vector), vector.getValueCount());
+               new VectorSchemaRoot(schema, Collections.singletonList(vector), vector.getValueCount());
```

Comment:
Did you mean to make this change?

Reply:
CC @danepitkin @davisusanibar I think it would help contributors if we used a formatter like google-java-format instead of peppering them with questions like this that checkstyle apparently doesn't handle

Reply:
No, I'll clean up all of these silly formatting issues.

Reply:
Sorry about the repeated nits on the formatting. I agree with @lidavidm's idea.

Reply:
> CC @danepitkin @davisusanibar I think it would help contributors if we used a formatter like google-java-format instead of peppering them with questions like this that checkstyle apparently doesn't handle

I am currently working on PR #39712 to enable the `google-java-format` by using the spotless plugin.